azure-communication-ui/chat/src/main/java/com/azure/android/communication/ui/chat/presentation/ui/view_model/ChatScreenViewModel.kt

```diff
@@ -0,0 +1,22 @@
+package com.azure.android.communication.ui.chat.presentation.ui.view_model
+
+import com.azure.android.communication.chat.models.ChatMessage
+
+import com.azure.android.communication.ui.chat.redux.state.ReduxState
+
+// View Model for the Chat Screen
+data class ChatScreenViewModel(val messages: List<ChatMessage>, val state: String, var buildCount: Int)
+
+// Internal counter for early debugging
+private var buildCount = 0
+
+// Methods to Build the Chat Screen View Model from the Store
+internal fun buildChatScreenViewModel(
+    state: ReduxState,
+    messages: List<ChatMessage>
```

Comment:
would it be possible to create an Class that we can inject instead of list of messages. 

As we have redux state and an non redux state. Tomorrow, adding anything to non redux state will help avoid adding more params to this class.

Reply:
Class will be MessageRepository

It will implement List<ChatMessage>